CHANGES.rst

```diff
@@ -2,12 +2,10 @@
  Changes
 =========
 
-
 1.13.1 (unreleased)
 ===================
 
-- Nothing changed yet.
-
+- ``StrippedValidTextLine`` should accept single character lines.
```

Comment:
"should" seems ambigous to me in the context of a change not. Do they now? Did they not before? What was the behoviour before and what is it now? 

I've been told to try to write change log entries [using the imperative voice](https://github.com/zopefoundation/ZODB/pull/170#pullrequestreview-51890045), and I think that helps reduce confusion. Something like "Fix ``StrippedValidTextLine`` to correctly recognize single character lines as stripped. Previously 'b' would have been rejected."

Reply:
Sounds good to me, thanks!